app/main.py

```diff
@@ -1,8 +1,46 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self,
+                 comfort_class: int,
+                 clean_mark: int,
+                 brand: str) -> None:
+
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self,
+                 distance_from_city_center: int,
+                 clean_power: int,
+                 average_rating: float,
+                 count_of_ratings: int) -> None:
+
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, cars: [Car]) -> float:
```

Comment:
```suggestion
    def serve_cars(self, cars: list[Car]) -> float:
```

Reply:
after changing a waring appears "Builtin 'list' cannot be parameterized directly "
![Screenshot from 2024-02-28 21-10-37](https://github.com/mate-academy/py-car-wash-station/assets/86102615/e8d0669a-1cde-4560-b8d7-4ba4fce92181)

Reply:
The cause of the problem may be old python interpritator version (3.8 i use)

Reply:
You can use List from `typing` package